uno-calculator.desktop

```diff
@@ -0,0 +1,85 @@
+[Desktop Entry]
+Name=Uno Calculator
+Exec=/usr/bin/flatpak run --command=Calculator.Skia.Gtk uno.platform.uno-calculator
```

Comment:
this is wrong. it should be the name of the executable.

also the destkop file should be managed upstream.

Reply:
same as the icon

Reply:
This is done now